src/wp-includes/css/custom-properties.css

```diff
@@ -150,6 +150,9 @@ body {
 	/* Darker borders */
 	--wp-admin--widgets-holder--border--hover: #787c82;
 	--wp-admin--widget-chooser--border: #8c8f94;
+
+	--wp-admin--health-tab-active--box-shadow: #3582c4;
```

Comment:
In the media.css PR, you've added the whole `box-shadow` property value, but here's it's just the color. We should pick one approach - I think it would be best to stick with the whole property value, what do you think?